pulsar-broker/src/main/java/com/yahoo/pulsar/broker/namespace/OwnershipCache.java

```diff
@@ -130,9 +127,16 @@
                             if (LOG.isDebugEnabled()) {
                                 LOG.debug("Successfully acquired zk lock on {}", namespaceBundleZNode);
                             }
+                            final OwnedBundle bundle = new OwnedBundle(
+                                    ServiceUnitZkUtils.suBundleFromPath(namespaceBundleZNode, bundleFactory));
+                            // update the version
+                            localZkCache.getZooKeeper().exists(path, false, (rc1, path1, ctx1, stat) -> {
```

Comment:
yes, and we never update this node so, it will be always 0. so, we can't depend on version so, how about if we can store `EphemeralOwner(long)` and before deleting we can get stats again verifies `EphemeralOwner` is matching or not. 
So, we can either not check-version (keep as it is) or verifies `EphemeralOwner` before deleting it.